SOPT-iOS/Projects/Core/Sources/Firebase/RemoteConfig/RemoteConfigType.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  RemoteConfigType.swift
+//  Core
+//
+//  Created by 강윤서 on 7/7/25.
+//  Copyright © 2025 SOPT-iOS. All rights reserved.
+//
+
+import Foundation
+
+public enum RemoteConfigType: String {
```

Comment:
```suggestion
public enum RemoteConfigKey: String {
```
Key라는 명칭이 더 적합할 것 같아요 : )